index.php

```diff
@@ -21,7 +22,31 @@
 
 <body>
 
-<h1>Website Template</h1>
+    <header class="header">
+        <h2>Home</h2>
+        <a href="#">Add book</a>
+    </header>
+
+
+    <?php
+    $db = new PDO('mysql:host=db; dbname=car-collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
You're probably going to need a DB connection elsewhere down the line, worth making a function that returns it and putting it into a separate file.